### PR TITLE
Updates DDF UI to latest version of ace

### PR DESCRIPTION
**Pull request by djblue**
#### Who is reviewing it? 

@Bdthomson 
@Lambeaux 

#### Select relevant component teams: 

@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@tbatie

#### How should this be tested?

CI build

#### Any background context you want to provide?

https://github.com/connexta/ace/pull/9

#### What are the relevant tickets?

Fixes: #4772

### CODE REVIEW

**Comment by djblue** [comment]
build now

File: ui/package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "react-dom": "16.4.2"
   },
   "devDependencies": {
-    "@connexta/ace": "git://github.com/connexta/ace.git#v0.0.7",
+    "@connexta/ace": "git://github.com/connexta/ace.git#20d216f6377802e2bc931331e3abf5dd8108b5c8",
```
Comment (willwill96) [inline ui/package.json]: ✏️ Seems strange to reference this by a commit number. Why not upgrade the version in ace and reference that?

**Comment by djblue** [comment]
build now

File: ui/package.json (hunk 1 above)
Comment (Lambeaux) [reply ui/package.json]: @djblue Could we get some more elaboration here on the problem that the hashes solve?

Comment (djblue) [reply ui/package.json]: With git hashes, I'm trying to strike the balance between the convenience snapshot versions and stability release versions.

**Comment by djblue** [comment]
build now

**Pull request merged**